src/test/resources/spring/gd-annotationConfig.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd">
+
+    <context:annotation-config/>
+
+    <bean id="gd" class="com.gooddata.GoodData">
+        <constructor-arg name="login" value="roman@gooddata.com" />
+        <constructor-arg name="password" value="supercoolsecret" />
```

Comment:
Should read `Roman1` according to https://github.com/martiner/gooddata-java/wiki/Code%20Examples#initialisation ;)